src/app/components/SubjectHeading/Search/SubjectHeadingSearch.jsx

```diff
@@ -126,6 +144,7 @@ class SubjectHeadingSearch extends React.Component {
               activeSuggestion={index === activeSuggestion}
               key={suggestion.uuid || suggestion.label}
               onClick={this.resetAutosuggest}
+              hidden={this.state.hidden}
```

Comment:
Instead of passing the hidden state, just have this whole `ul` not render. On line 72, you could do something like `{ !hidden && suggestionsListComponent}`. Otherwise, an empty `ul` is rendered and effects the styling.

Reply:
Good point. Not sure I understand the comment about line 72 but I've changed the logic to not render any `ul`.